action.yml

```diff
@@ -72,14 +91,22 @@ runs:
         VERSION: ${{ steps.latest.outputs.release || inputs.cli-version }}
         OS: ${{ steps.os.outputs.target }}
         ARCH: ${{ steps.arch.outputs.target }}
+        MAJOR_VERSION: ${{ steps.version.outputs.cli_major }}
+        CLI_BIN: ${{ steps.version.outputs.cli_bin }}
       run: |
-        curl -fsSL "http://github.com/${REPO}/releases/download/${VERSION}/balena-cli-${VERSION}-${OS}-${ARCH}-standalone.zip" -o balena-cli.zip
-        unzip balena-cli.zip -d .
+        if [ "${MAJOR_VERSION}" -lt 22 ]; then
+          curl -fsSL "http://github.com/${REPO}/releases/download/${VERSION}/balena-cli-${VERSION}-${OS}-${ARCH}-standalone.zip" -o balena-cli.zip
+          unzip balena-cli.zip -d .
+        else
+          curl -fsSL "http://github.com/${REPO}/releases/download/${VERSION}/balena-cli-${VERSION}-${OS}-${ARCH}-standalone.tar.gz" -o balena-cli.tar.gz
+          tar -xzf balena-cli.tar.gz
+        fi
+        echo "${PWD}/${CLI_BIN}" >> "$GITHUB_PATH"
```

Comment:
Not sure if we should do that or rather just expose the bin folder as an output and leave the customer to append to the path if they wish